lib/crypto/packaged.py

```diff
@@ -70,6 +70,8 @@ def call_signing(file_obj, endpoint):
         response = requests.post(
             endpoint,
             timeout=settings.SIGNING_SERVER_TIMEOUT,
+            # The guid must be 64chars or less: see bug 1201176.
+            # This should be enforced during the add-on submission.
```

Comment:
Not yet, I don't think so, we haven't decided what we're doing with the current add-ons with too long GUIDs. In fact, I think I'm going to remove this comment, which isn't related to the bug this PR (or rather, the new PR #717) is addressing: preventing the submission of new add-ons with long GUIDs.